types/ronilaukkarinen__gulp-stylelint/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "private": true,
     "dependencies": {
-        "stylelint": "^14.0.1"
+        "stylelint": "*"
```

Comment:
I believe this is not the best approach, exposes vector of attack.
Can this be more specific?
https://github.com/ronilaukkarinen/gulp-stylelint/blob/master/package.json#L35

Reply:
Thanks. The file is updated.